app/views/imports/_form_status.html.erb

```diff
@@ -0,0 +1,35 @@
+<div class="card shadow border-0">
+  <div class="card-header-success">
+    <h5 class="card-title">Import</h5>
+  </div>
+
+  <div class="card-body row">
+    <div class="col-12 mb-4 prx-field-group">
+      <div class="form-floating">
+        <%= form.select :feed_source, ["RSS"] %>
+        <%= form.label :feed_source, "Feed source" %>
+      </div>
+
+      <button type="button" class="prx-btn-help" data-bs-toggle="popover" data-bs-content="<%= "help" %>"><span class="material-icons">help</span></button>
+    </div>
+
+    <div class="col-12 mb-4 prx-field-group">
+      <div class="form-floating">
+        <%= form.text_field :url %>
+        <%= form.label :url, "RSS Feed URL" %>
+      </div>
+
+      <button type="button" class="prx-btn-help" data-bs-toggle="popover" data-bs-content="<%= "help" %>"><span class="material-icons">help</span></button>
+    </div>
+
+  </div>
+
+  <div class="card-footer d-flex-align-items-center justify-content-between">
+
+    <div class="col-12 mb-4">
+      <%= form.hidden_field :account_id, value: @podcast.account_id %>
+      <%= form.hidden_field :podcast_id, value: @podcast.id %>
```

Comment:
these `hidden_field`s i think were defined in the `index` action because i think they are necessary for a valid `PodcastImport`? is there a better way to handle this?

Reply:
So I think you should drop `account_id` ... it's just not needed.

And `:podcast_id` shouldn't be a form field.  Once you added a `has_many :podcast_imports` in `podcast.rb`... in your controller you can just do a more standard:

```
@import = @podcast.podcast_imports.new(import_params)
```